Build Error: vector_map_server/GetWhiteLine.h: No such file or directory

Hi,

I'm having a little trouble getting Autoware installed on an Ubuntu 16.04 VM with ROS Kinetic. I see the following:

```shell
[ 83%] Building CXX object data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/nodes/vector_map_client/vector_map_client.cpp.o
/home/carnd/Autoware/ros/src/data/packages/vector_map_server/nodes/vector_map_client/vector_map_client.cpp:36:44: fatal error: vector_map_server/GetWhiteLine.h: No such file or directory
compilation terminated.
data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/build.make:62: recipe for target 'data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/nodes/vector_map_client/vector_map_client.cpp.o' failed
make[2]: *** [data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/nodes/vector_map_client/vector_map_client.cpp.o] Error 1
CMakeFiles/Makefile2:19193: recipe for target 'data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/all' failed
make[1]: *** [data/packages/vector_map_server/CMakeFiles/vector_map_client.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
Invoking "make -j1 -l1" failed
```

I've tried `catkin_make --pkg vector_map_server` and see the same.

I also see the following warnings:
```shell
CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'ssh2' but neither 'ssh2_INCLUDE_DIRS' nor
  'ssh2_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  data/packages/pos_db/CMakeLists.txt:17 (catkin_package)


CUDA_TOOLKIT_ROOT_DIR not found or specified
CUDA is not installed. 'cv_tracker' package is not built.
'FastRCNN/Caffe' are not installed. 'rcnn_msr' will not be built.
'SSD/Caffe' is not installed. 'ssd_unc' will not be built.
'Darknet' was not found. 'yolo2_wa' will not be built.
CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'system_lib' but neither
  'system_lib_INCLUDE_DIRS' nor 'system_lib_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  computing/planning/motion/packages/op_simulator/CMakeLists.txt:21 (catkin_package)


CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'system_lib' but neither
  'system_lib_INCLUDE_DIRS' nor 'system_lib_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  computing/planning/motion/packages/op_simulator_perception/CMakeLists.txt:20 (catkin_package)


CUDA_TOOLKIT_ROOT_DIR not found or specified
CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'yaml-cpp' but neither 'yaml-cpp_INCLUDE_DIRS'
  nor 'yaml-cpp_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  sensing/drivers/lidar/packages/velodyne/velodyne_pointcloud/CMakeLists.txt:39 (catkin_package)


CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'python-yaml' but neither
  'python-yaml_INCLUDE_DIRS' nor 'python-yaml_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  sensing/drivers/lidar/packages/velodyne/velodyne_pointcloud/CMakeLists.txt:39 (catkin_package)


** WARNING ** io features related to pcap will be disabled
** WARNING ** io features related to png will be disabled
'BGGAPI' is not installed. 'vlg22c_cam' will not be built.
CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'curl' but neither 'curl_INCLUDE_DIRS' nor
  'curl_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  data/packages/map_file/CMakeLists.txt:21 (catkin_package)


CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'system_lib' but neither
  'system_lib_INCLUDE_DIRS' nor 'system_lib_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  computing/planning/motion/packages/ff_waypoint_follower/CMakeLists.txt:21 (catkin_package)


'SSD/Caffe' is not installed. 'region_tlr_ssd' will not be built.
CMake Warning at /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:166 (message):
  catkin_package() DEPENDS on 'system_lib' but neither
  'system_lib_INCLUDE_DIRS' nor 'system_lib_LIBRARIES' is defined.
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  computing/planning/mission/packages/way_planner/CMakeLists.txt:27 (catkin_package)

CMake Warning (dev) at computing/perception/detection/packages/cv_tracker/CMakeLists.txt:230 (ADD_DEPENDENCIES):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "libdpm_ocv_generate_messages_cpp" of target
  "klt_track" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/lidar_tracker/CMakeLists.txt:289 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "lidar_tracker_generate_messages_cpp" of target
  "euclidean_cluster" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:182 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "road_wizard_generate_messages_cpp" of target
  "region_tlr" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "<<<<<<<" of target "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "=======" of target "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target ">>>>>>>" of target "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "HEAD" of target "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "fe3055867d3a8cd8990ab0541b7a69538eb1c322" of target
  "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "road_wizard_generate_messages_cpp" of target
  "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "runtime_manager_generate_messages_cpp" of target
  "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at computing/perception/detection/packages/road_wizard/CMakeLists.txt:99 (add_dependencies):
  Policy CMP0046 is not set: Error on non-existent dependency in
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.
  Use the cmake_policy command to set the policy and suppress this warning.

  The dependency target "waypoint_follower_generate_messages_cpp" of target
  "feat_proj" does not exist.
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Thanks,

Stephen

--- a/ros/src/data/packages/vector_map_server/CMakeLists.txt
+++ b/ros/src/data/packages/vector_map_server/CMakeLists.txt
@@ -66,11 +66,13 @@ include_directories(
 add_executable(vector_map_server nodes/vector_map_server/vector_map_server.cpp)
 target_link_libraries(vector_map_server ${catkin_LIBRARIES})
 add_dependencies(vector_map_server
+  vector_map_server_generate_messages_cpp
   ${catkin_EXPORTED_TARGETS}
 )
 
 add_executable(vector_map_client nodes/vector_map_client/vector_map_client.cpp)
 target_link_libraries(vector_map_client ${catkin_LIBRARIES})
 add_dependencies(vector_map_client
+  vector_map_server_generate_messages_cpp
   ${catkin_EXPORTED_TARGETS}
 )

